hoodie-client/src/test/java/com/uber/hoodie/common/HoodieTestDataGenerator.java

```diff
@@ -221,6 +223,22 @@ public static void createSavepointFile(String basePath, String commitTime) throw
       os.close();
     }
   }
+  
+  public List<HoodieRecord> generateInsertsSpecificRecord(int n) throws IOException {
```

Comment:
lets stick this as another static helper under `HoodieClientTestUtils` and leave this file working with the flat schema used everywhere..

Reply:
I do acknowledge the test helpers and frameworks, need a lot of love.. cc @bvaradar

Reply:
If I move them in HoodieClientTestUtils I have to create partitions and ran variable I already have in HoodieTestDataGenerator and when I look in HoodieClientTestUtils is read utilities methods

Reply:
Hmmm. I ll bite.. given a lot of this is coming from how its all laid out inherently.